src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestChannel.java

```diff
@@ -253,4 +254,11 @@ InsertValidationResponse insertRows(
    */
   @Nullable
   String getLatestCommittedOffsetToken();
+
+  /**
+   * Gets the table schema associated with this channel
+   *
+   * @return map representing Column Name -> Column Properties
+   */
+  Map<String, ColumnProperties> getTableSchema();
```

Comment:
Could you make it clear in the comment that this is not update to date schema?

Reply:
Yeah good call out.